HelloMixpanel/HelloMixpanelTests/MixpanelOptOutTests.m

```diff
@@ -191,8 +179,14 @@ - (void)testOptOutWillSkipFlushPeople
     
     NSMutableArray *peopleQueue = [NSMutableArray arrayWithArray:self.mixpanel.peopleQueue];
     [self.mixpanel optOutTracking];
-    [self waitForMixpanelQueues];
```

Comment:
i think it's fine to wait here since it's more necessary. code will be cleaner too